process/sync/storageBootstrap/baseStorageBootstrapper.go

```diff
@@ -71,7 +71,10 @@ func (st *storageBootstrapper) loadBlocks() error {
 	round := st.bootStorer.GetHighestRound()
 	if round == 0 {
 		log.Debug("Load blocks does nothing as start from genesis")
-		return nil
+		err = st.bootStorer.SaveLastRound(0)
```

Comment:
I think this is not necessary as anyway 0 is the default value. Also you can remove lines 73 and 75